obsah/__init__.py

```diff
@@ -242,10 +243,26 @@ def allow_extra_vars() -> bool:
         """
         Whether to allow --extra-vars parameter to be automatically added
         """
-        if value := os.environ.get('OBSAH_ALLOW_EXTRA_VARS') is not None:
+        if (value := os.environ.get('OBSAH_ALLOW_EXTRA_VARS')) is not None:
             return value.lower() in ['true', '1']
         return True
 
+    @staticmethod
+    def persist_params():
+        """
+        Whether or not to persist parameters
+        """
+        if (value := os.environ.get('OBSAH_PERSIST_PARAMS')) is not None:
+            return value.lower() in ['true', '1']
+        return False
+
+    @staticmethod
+    def persist_path():
+        """
+        Where to persist parameters to
+        """
+        return os.environ.get('OBSAH_PERSIST_PATH', '/var/lib/obsah/parameters.yaml')
```

Comment:
I'd lean to introducing a state directory and make this `state_dir / 'parameters.yaml'`. This may be a bit of bike shedding, but perhaps respect `XDG_STATE_HOME` so it can work out of the box in test setups as well. Then in rop packaging we should override it with `OBSAH_STATE_DIR=/var/lib/rop`